src/main/java/org/sonar/plugins/github/MarkDownUtils.java

```diff
@@ -44,6 +44,7 @@ public MarkDownUtils(Settings settings) {
     if (!baseUrl.endsWith("/")) {
       baseUrl += "/";
     }
+    System.out.println("Create issue");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106)